lib/llvmopencl/SubCFGFormation.cc

```diff
@@ -267,10 +283,17 @@ getLocalSizeValues(llvm::Function &F, llvm::ArrayRef<unsigned long> LocalSizes,
 
       if (I->getParent() != &F.getEntryBlock()) {
         // must be in entry block. move.
+// TODO for some reason, moveAfter(InsertionPoint) exists in the header, but
```

Comment:
Leftover?

Reply:
"leftover" as in, an actual remaining unsolved problem. For now it works fine with the old code, because passing of Instruction instead of InsertionPoint is only deprecated (not removed), but they will likely remove it in future LLVM version.